WmgmtBin/schema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$schemaVersion": "0.0.1",
+  "$id": "https://smart-data-models.github.io/dataModel.WasteManagement/WmgmtBin/schema.json",
+  "title": "Smart Data models -WmgmtBin schema",
+  "description": "Data model for solid waste management bins.",
+  "type": "object",
+  "required": [
+    "id",
+    "type"
+  ],
+  "allOf": [
+    {
+      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
+    },
+    {
+      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+    },
+    {
+      "properties": {
+        "binCapacity": {
+          "type": "number",
+          "description": "Property. Model:'https://schema.org/Number'. Total capacity in terms of the volume of waste the bin can hold."
+        },
+        "binColor": {
+          "type": "string",
+          "description": "Property. Model:'https://schema.org/Text'. Color of the bin. Could be used for indicating the type of waste. The color coding should follow the conventions applicable to the geographical area the bins are located."
+        },
+        "binClearedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Property. Model:'https://schema.org/Text'. Time when the bin was last cleared."
+        },
+        "wardID": {
+          "type": "string",
+          "description": "Property. Model:'https://schema.org/Text. Ward ID of the entity corresponding to this observation."
```

Comment:
warId?
is it a relationship?

Reply:
wardID is a number assigned to a ward (section of a city). It is not considered a relationship